lib/acts_as_taggable_on_mongoid/taggable/tag_type_definition/changeable.rb

```diff
@@ -11,16 +11,32 @@ class TagTypeDefinition
       #   * tag_list_will_change!
       #   * tag_list_changed_from_default?
       #   * tag_list_was
+      #   * tagger_tag_list_was
+      #   * tag_lists_was
       #   * reset_tag_list!
       #   * reset_tag_list_to_default!
       module Changeable
+        def default_tagger_tag_list(taggable)
+          list = ActsAsTaggableOnMongoid::TaggerTagList.new(self, nil)
+
+          list_default              = default.dup
+          list_default.taggable     = taggable
```

Comment:
Refers to 'list_default' more than self (maybe move it to another class?) (FeatureEnvy)